SiG/laser-util-sig.py

```diff
@@ -557,7 +559,12 @@ def get_battery_state(self):
         return "%.2f%% (%s)" % (perc, "charging" if charging else "discharging")
 
     def dump(self, label):
+        pns = self.get_firmware_partnumbers()
+        arm_pn = pns[:6]
+        fpga_pn = pns[6:]
```

Comment:
I get that arm_pn will definitely output 6 bytes, but I'm curious about fpga_pn.  Is there something in the USB response packet which states that only 12 bytes are being returned, of a potential buffer of 64 bytes?

This isn't a bug report or complaint, I'm just figuring that the number "12" needs to appear somewhere in either the software or the firmware, and I'm curious how the "payload length" gets communicated from ARM to host.